showtime-xyz/showtime-frontend
# refactor(SHOW2-762): fixed login modal screen layout

# Why

Since the #1035 the login screen on mobile is broken, especially when it comes to the keyboard handling.

# How

https://user-images.githubusercontent.com/4061838/166563348-84dd3203-2e4a-4583-b8e8-a3d7ea511d20.mov


# Test Plan

- launch the app
- make sure that you are logged out
- tap on sign in button
  - a modal screen should appear
- check phone number & email text input
  - content height should resize based on keyboard appearance
